CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 ### Changed
 - Update cm42-central-support version to save delivered_at
 
+### Added
+- Highlights to release stories if the release date is compromised.
```

Comment:
maybe `Highlights the release stories if the release date is compromised.`

Reply:
That does sounds good.
The idea with the `to` is to reading the `added` first. thus becoming more like:
Added highlights to the release stories if the release dade is compromised.
What do you think?